computervision/src/main/java/ComputerVisionInAction.kt

```diff
@@ -19,14 +19,13 @@ internal class ComputerVisionInAction(private val cvPortal: SensorPortal) : Comp
     private fun cvTokenize(rawData: String): Vision? {
         try {
             val raw = JSONObject(rawData)
-            val list: List<Any> = listOf<Any>(raw.getJSONArray("value")) //[[......]]
 
             return when (raw.getInt("type")) {
                 OCR -> Vision.OCR(raw.getString("value"))
-                scenes -> Vision.Scenery(raw.getString("value"))
-                prettyFaces -> Vision.Facial(raw.getString("value"))
-                emotions -> Vision.Emotion(raw.getString("value"))
-                objects -> Vision.ObjectDetection(list)
+                SCENES -> Vision.Scenery(raw.getString("value"))
+                PRETTY_FACES -> Vision.Facial(raw.getString("value"))
+                EMOTIONS -> Vision.Emotion(raw.getString("value"))
+                OBJECTS -> Vision.ObjectDetection(listOf<Any>(raw.getJSONArray("value"))) //[[......]])
```

Comment:
Perfect. We will change that `Any` when the specs are completely finished. But this will work for now.